plugins/woocommerce/tests/e2e-pw/tests/admin-marketing/overview.spec.js

```diff
@@ -15,40 +15,44 @@ test.describe( 'Marketing page', () => {
 		).toBeVisible();
 	} );
 
-	test( 'Marketing Overview page have relevant content', async ( {
-		page,
-	} ) => {
-		// Go to the Marketing page.
-		await page.goto( 'wp-admin/admin.php?page=wc-admin&path=%2Fmarketing' );
+	test(
+		'Marketing Overview page have relevant content',
+		{ tag: '@skip-on-default-wpcom' },
```

Comment:
There's no "Discover more marketing tools" and related content on the WPCOM site.